extensions/authentication-passport/README.md

```diff
@@ -0,0 +1,245 @@
+# Passport Strategy Adapter
+
+_Important: We strongly recommend that users learn LoopBack's
+[authentication system](https://loopback.io/doc/en/lb4/Loopback-component-authentication.html)
+before using this module._
+
+This is an adapter module created for plugging in
+[`passport`](https://www.npmjs.com/package/passport) based strategies to the
+authentication system in `@loopback/authentication@3.x`.
```

Comment:
I'm wondering whether we want to specify the version `@loopback/authentication@3.x`.  But feel free to ignore.

Reply:
@dhmlau I feel it's important, since `@loopback/authentication@1.x` is not compatible with this module, and 2.x imports a different `UserProfile`. Good question though :)

Reply:
Good point. I was more coming from the perspective that when we're going to have 4.x 5.x, are we going to make changes every time. :) . But it's a minor issue, we can figure it out when it happens.